### PR TITLE
List -Framework

### PR DESCRIPTION
**Pull request by katakampranavi**
Issue #9 
Feat: Added the list framework page.
! [Project Snapshot]
https://drive.google.com/file/d/14ACD0w0cf1toK6lRAMG-Rgcqwh4LsK20/view?usp=sharing

### CODE REVIEW

File: sunbird-framework-management-tool/package.json [hunk 1]
```diff
@@ -5,13 +5,20 @@
...
-    "@mui/material": "^5.14.0",
+    "@mui/icons-material": "^5.14.1",
+    "@mui/material": "^5.14.2",
+    "@mui/styled-engine-sc": "^5.12.0",
+    "@mui/x-data-grid": "^6.10.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.4.0",
+    "http-proxy-middleware": "^2.0.6",
```
Comment (HarishGangula) [inline sunbird-framework-management-tool/package.json:16]: why this package is required

File: sunbird-framework-management-tool/package.json [hunk 2]
```diff
@@ -5,13 +5,20 @@
...
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.14.2",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.11",
```
Comment (HarishGangula) [inline sunbird-framework-management-tool/package.json]: where are we using the styled components

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: sunbird-framework-management-tool/package.json (hunk 1 above)
Comment (katakampranavi) [reply sunbird-framework-management-tool/package.json:16]: @HarishGangula Yesterday when I did, I got Cors's error to remove that i downloaded that

File: sunbird-framework-management-tool/package.json (hunk 2 above)
Comment (katakampranavi) [reply sunbird-framework-management-tool/package.json]: @HarishGangula i tried to use but I haven't used it

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**